code/game/objects/items/reagent_containers/hypospray.dm

```diff
@@ -138,6 +133,17 @@
 			playsound(user.loc, 'sound/weapons/thudswoosh.ogg', 25, 1, 7)
 			return FALSE
 
+		if(!M.can_inject(user, TRUE, user.zone_selected, TRUE))
```

Comment:
Why the code duplication here? (same check a few lines up)

Reply:
Distracted there, now fixed.